src/libsync/propagatedownload.cpp

```diff
@@ -940,20 +947,20 @@ void PropagateDownloadFile::downloadFinished()
             return;
         }
     }
+    // If the file is locked, we want to retry this sync when it
+    // becomes available again
+    if (FileSystem::isFileLocked(fn, FileSystem::LockMode::Exclusive)) {
+        emit propagator()->seenLockedFile(fn, FileSystem::LockMode::Exclusive);
+        done(SyncFileItem::SoftError, tr("%1 is currently in use").arg(fn));
+        return;
+    }
```

Comment:
I wonder who/what removes the tmp file of the download that is stopped here. But there are already other checks that interrupt the download, so that is probably handled elsewhere. Otherwise that would leave tmp files behind, no?

Reply:
I'd doubt it but that something for a different review :D